core/app_modules/inventory.js

```diff
@@ -287,8 +287,9 @@ Object.defineProperty(inventory, 'checked_out', {
                 '( SELECT array_agg(l.name) as languages FROM languages l ' +
                 'JOIN languages_items li ON li.language_code = l.code AND ' +
                 'li.inventory_call = i.call ), ' +
-                '( SELECT array_agg(m.medium) as media FROM media_items m ' +
-                'WHERE m.call = i.call ) ' +
+                '( SELECT array_agg(m.code) as media FROM media m ' +
```

Comment:
It might be nice to also include the media type in a hover or something similar. @stevethomas19 what do you think?  Looks good to me though.